compiler/one-cmds/utils.py

```diff
@@ -81,18 +81,19 @@ class _CONSTANT:
 _CONSTANT = _CONSTANT()
 
 
-def _add_default_arg(parser):
+def _add_default_arg(parser, support_config=True):
```

Comment:
I've little understanding about this but seems not related with adding `one-profile` tool. (`one-profile` tool may use this feature)
if so, please separate this change to another PR.

Reply:
`_add_default_arg` seems to be called from all `one-xxx` tools. if you make this default as `True` all the commands will have `--config` and `--section` options, which I don't see other tools are ready to use this.

I'm my understanding is incorrect, please let me know.

Reply:
Okay I see. :) I will make it as another PR.

Reply:
This is probably an attempt to remove the `-C` option. I think it's an unnecessary part, as I commented before.